packages/block-editor/src/components/block-preview/auto.js

```diff
@@ -59,7 +62,14 @@ function ScaledBlockPreview( {
 	// Initialize on render instead of module top level, to avoid circular dependency issues.
 	MemoizedBlockList = MemoizedBlockList || pure( BlockList );
 
+	viewportWidth = viewportWidth / ( __experimentalScale || 1 );
```

Comment:
This is quite confusing to me. We already calculate a `scale` (right below) and we still pass a new `scale` that takes part of the calculation. What was the need for this?

Reply:
scale prop `__experimentalScale` is an alternative to reduce `viewportWidth` by certain percentage. line below takes scaled viewportWidth for calculation.

Reply:
change reverted.